src/Components/ADT3DViewer/ADT3DViewer.tsx

```diff
@@ -725,12 +726,18 @@ const ADT3DViewerBase: React.FC<IADT3DViewerProps> = ({
                             : undefined
                     }}
                 />
-                {/* Mode & layers */}
+                {/* Header controls */}
                 <Stack
                     horizontal
                     styles={classNames.subComponentStyles.headerStack}
                     tokens={{ childrenGap: 8 }}
                 >
+                    <SceneRefreshButton
+                        isRefreshing={isLoading}
+                        lastRefreshTimeInMs={lastRefreshTime}
+                        refreshFrequency={nextRefreshTime - lastRefreshTime}
```

Comment:
shouldn't this "refresh frequency" a static number? can't we do this calculation internally?

Reply:
no need then